### DISCUSSION TITLE
[bugfix/64259] Bracket types are breaking the code in utils.d.ts []

### DISCUSSION THREAD

**Discussion by bbereczky-wecan** [DefinitelyTyped/DefinitelyTyped#64259] [Issues with a @types package]
In `MIMEParams` / `entries()` the return type `IterableIterator` is wrongly typed. 
![Screenshot from 2023-02-07 16-30-05](https://user-images.githubusercontent.com/104906258/217289018-98726b86-0ac5-47f9-a209-b06bb825fa51.png)

[a comment by an automated account is not shown]

**Reply by peterblazejewicz** [reply]
@bbereczky-wecan what's the version of TypeScript in your current setup? (I understand a problem with the unexpected outcome as a result from @types/node updates)

**Reply by Semigradsky** [reply]
It looks like `TypeScript 3.x`.

`Labeled Tuple Elements` feature was added in 4.0 - https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#labeled-tuple-elements
